hermeto/core/package_managers/python/uv/models.py

```
@@ -0,0 +1,564 @@
+    ``location`` is the URL of the one file the package resolves to.
+    """
+
+    kind: Literal["url"]
+    location: str
+
+    def purl(
+        self,
+        name: str,
+        version: str | None,
+        checksum: str | None,
+        **kwargs: Any,  # noqa: ARG002
```

Ruff (0.16.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)